src/blocks/rofication.rs

```diff
@@ -148,26 +145,20 @@ fn rofication_status(socket_path: &str) -> Result<RotificationStatus> {
     };
 
     // Request count
-    match stream.write(b"num\n") {
-        Err(_) => {
-            return Err(BlockError(
-                "rofication".to_string(),
-                "Failed to write to socket".to_string(),
-            ))
-        }
-        Ok(_) => {}
+    if stream.write(b"num\n").is_err() {
```

Comment:
You can also use `stream.write(b"num\n").error("rofication", "Failed to write to socket")?;`

Reply:
Thanks @MaxVerevkin  :-) Done, applied in other places as well.